spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/config/BenchmarkRunnerConfig.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.spark.benchmark.config;
+
+import static com.palantir.logsafe.Preconditions.checkArgument;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.palantir.spark.benchmark.immutables.ImmutablesConfigStyle;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@ImmutablesConfigStyle
+@JsonDeserialize(as = ImmutableBenchmarkRunnerConfig.class)
+public interface BenchmarkRunnerConfig {
+    ObjectMapper MAPPER = new ObjectMapper(new YAMLFactory()).registerModules(new Jdk8Module(), new GuavaModule());
```

Comment:
we should just put this inside the `parse` method, rather than it being a public-static-final field (it's presumably only used once per JVM lifetime anyway)